PowerEditor/installer/nativeLang/russian.xml

```diff
@@ -1587,6 +1599,14 @@ Updated to v8.4.6:
 			<contextMenu-styleOneToken              value="Пометить только Одно"/>
 			<contextMenu-clearStyle                 value="Очистить Пометки"/>
 			<contextMenu-PluginCommands             value="Команды Плагина"/>
+			<enable-disable-largeFileRestriction-tip    value="Переключение &quot;Ограничение на Большой Файл&quot; вступит в силу только после закрытия и повторного открытия файла."/>
+			<change-largeFileRestriction_fileLength-tip value="Изменение размера файла вступит в силу только после закрытия и повторного открытия файла."/>
+			<largeFileRestriction_wordWrap-tip          value="Включение &quot;Выключить Перенос строк глобально&quot; опция деактивирует Перенос Строк для всех открытых документов после открытия большого файла. Вы можете повторно активировать Перенос Строки вручную."/>
+			<largeFileRestriction-tip                   value="Некоторые функции могут снизить производительность при работе с большими файлами. Эти функции могут автоматически отключаться при открытии большого файла. Вы можете настроить их здесь.
+ПРИМЕЧАНИЕ:
+1. При изменение данных параметров, потребуется повторное открытие текущих больших файлов, чтобы изменения вступили в силу.
+
+2. Если &quot;Выключить Перенос строк глобально&quot; активно и вы открываете большой файл, то &quot;Перенос строк&quot; будет отключен для всех файлов. Вы можете повторно включить его через меню &quot;Вид-&gt;Перенос строк&quot;."/>
```

Comment:
Could you please make PR matched with latest updated english.xml? 
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/12404